internal/plugin.go

```diff
@@ -386,10 +386,26 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 
 	for i := range p.Policies {
 		policy := &p.Policies[i]
+		if len(policy.PolicyAnnotations) == 0 {
+			annotations := make(map[string]string)
+			for k, v := range p.PolicyDefaults.PolicyAnnotations {
+				annotations[k] = v
+			}
+			policy.PolicyAnnotations = annotations
+		}
+
 		if policy.Categories == nil {
 			policy.Categories = p.PolicyDefaults.Categories
 		}
 
+		if policy.Standards == nil {
```

Comment:
Optional: This change is unnecessary and it'd be better if this and the controls section were left like they were originally in order to keep the Git history clean.